### PR TITLE
UIREQ-879: UI tests replacement with RTL/Jest for src/PositionLink.js

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
UI tests replacement with RTL/Jest for src/PositionLink.js

## Refs
https://issues.folio.org/browse/UIREQ-879

### CODE REVIEW

File: src/PositionLink.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+      it('should render link position', () => {
+        expect(screen.getByText(`${normalConfig.request.position} (${labelIds.requests})`)).toBeInTheDocument();
+      });
+
+      it('should render view requests in queue', () => {
+        expect(screen.getByText(labelIds.viewRequestsInQueue)).toBeInTheDocument();
+      });
+
+      it('should render view requests in queue', () => {
```
Comment (artem-blazhko) [inline src/PositionLink.test.js:71]: Could you use unique test names?
In the current situation, only one test will be executed twice and another one will be ignored.

Comment (Dmitriy-Litvinenko) [reply src/PositionLink.test.js]: Fixed

**Review by artem-blazhko — APPROVED** [review]

File: src/PositionLink.test.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        expect(screen.getByText(`${normalConfig.request.position} (${labelIds.requests})`)).toBeInTheDocument();
+      });
+
+      it('should render view requests in queue', () => {
+        expect(screen.getByText(labelIds.viewRequestsInQueue)).toBeInTheDocument();
+      });
+
+      it('should create correct href', () => {
+        const filters = 'requestStatus.Open - Not yet filled,requestStatus.Open - Awaiting pickup,requestStatus.Open - Awaiting delivery,requestStatus.Open - In transit';
+        const href = `/requests?filters=${filters}&query=${isTlrEnabled ? 'instanceId' : 'itemId'}&sort=Request Date`;
```
Comment (BogdanDenis) [inline src/PositionLink.test.js:73]: IMO tests shouldn't have any logic at all. As soon as tests contain some kind of logic it's much more difficult to believe that tests themselves don't contain any bugs
I know that it means some duplication, but then you have more confidence in tests
If you think that it's ok to have logic in this case - please rename `positionLink` to something like `runSuiteWithParam` so it's easier to understand what's happening

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/PositionLink.test.js (hunk 2 above)
Comment (Dmitriy-Litvinenko) [reply src/PositionLink.test.js:73]: Hello @BogdanDenis Thank you. Could you please re-review?

**Pull request merged**